packs/unctl-1.0.0/values.yaml

```diff
@@ -0,0 +1,76 @@
+pack:
+  # The namespace (on the target cluster) to install this chart
+  # When not found, a new namespace will be created
+  namespace: "monitoring"
+  
+  content:
+    images:
+      - image: public.ecr.aws/unskript/unctl:1.0.0
+      - image: public.ecr.aws/unskript/unctl-frontend:1.0.0
+
+charts:
+  frontend: 
```

Comment:
frontend: and the content following that should be child of the unctl:
charts:
   unctl:
      frontend: 
and so on.
Structure to be followed is have a child for each chart under the `charts` by `name of the chart` and have the values.yaml content under it. This way, it is possible to support multiple charts in a pack

Reply:
in the current configuration it is not really possible to split them into child charts as for eg. `frontend` is just additional container under same deployment in the chart, so all resources are pretty much dependant.

Reply:
No need to split into child charts. just that move these contents under a parent chart, which in this case is unctl. This hierarchy is needed for the changes we make in packs' values.yaml to take effect.

Reply:
done